hypothesis/abstact.js

```diff
@@ -45,3 +50,27 @@ AbstactStudentT.prototype.valid = function () {
 AbstactStudentT.prototype.freedom = function () {
   return this._df;
 }
+
+function inverseStudent(k, p) {
```

Comment:
This is inspired from cephes implementation: https://github.com/jeremybarnes/cephes/blob/60f27df395b8322c2da22c83751a2366b82d50d1/cprob/stdtr.c#L187-L225

Reply:
A couple of things to note about cephes: 

1) The official version is here https://www.netlib.org/cephes/
2) Be careful with copying from strange sources because the code is unlicensed. But scipy and me (for node-cephes) had to ask the prof. Moshier for a BSD license.
3) There was an update a few years ago, properly not to stdtr or stdtri, but you never know.